indexer/services/ender/src/helpers/postgres/postgres-functions.ts

```diff
@@ -93,10 +93,15 @@ const MAIN_SCRIPTS: string[] = [
   'dydx_block_processor.sql',
 ];
 
+const BACKFILL_SCRIPTS: string[] = [
```

Comment:
I don't think we need to add it here.

Reply:
hmmm - for my understanding, how do these sql functions get created?